src/mote/templates/mainpage.html

```diff
@@ -33,6 +33,7 @@
         <link rel="icon" href="{{ url_for('static', filename='imgs/logoteal.png') }}" sizes="16x16">
         <script src="{{ url_for('static', filename='jscn/fragment.js') }}"></script>
         <script src="{{ url_for('static', filename='jscn/easy.qrcode.min.js') }}"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/4.3.2/socket.io.js"></script>
```

Comment:
Please download and store the JavaScript library in `src/mote/static/jscn/` and reference it from there - just like we're doing for other such JavaScript dependencies.